queries.sql

```diff
@@ -1,3 +1,41 @@
--- /*Queries that provide answers to the questions from all projects.*/
+-- Find all animals whose name ends in "mon".
+SELECT * 
+FROM animals
+WHERE name LIKE '%mon'
 
--- SELECT * from animals WHERE name =   'Luna';
+-- List the name of all animals born between 2016 and 2019.
+SELECT name
+FROM animals
+WHERE (date_of_birth) between '2016-01-01' and '2019-12-31'
+
+-- List the name of all animals that are neutered and have less than 3 escape attempts.
+
+SELECT name
+FROM animals
+WHERE neutered=true and escape_attempts <3
+
+-- List the date of birth of all animals named either "Agumon" or "Pikachu".
+
+SELECT date_of_birth,name
+FROM animals
+WHERE name='Agumon'or name='pikachu'
+
+-- List name and escape attempts of animals that weigh more than 10.5kg
+SELECT name, escape_attempts
+From animals
+where weight_kg >10.5
```

Comment:
- [ OPTIONAL ] I think it would be great if you add quotes to the value since each value surronded with quotes so kindly add quotes suuround 10.5 value

> WHERE weight_kg > '10.5';